tcl/return.tcl

```diff
@@ -50,10 +50,10 @@ proc qc::return2client { args } {
     # Content-Disposition
     if { $download } {
         # Download payload as an attachment
-        default content-disposition "attachment; filename=$filename"
+        default content-disposition "attachment; \"filename=$filename\""
```

Comment:
http://stackoverflow.com/questions/93551/how-to-encode-the-filename-parameter-of-content-disposition-header-in-http

From the looks of it, that depends on which browser we are returning to.

I suspect that the best solution would simply be to white-list allowed filename characters, and strip out the rest.